stdlib/filepath/src/main/ballerina/src/filepath/file_path.bal

```diff
@@ -216,8 +237,11 @@ public function normalize(string path) returns string|Error {
 }
 
 # Splits a list of paths joined by the OS-specific Path Separator.
```

Comment:
```suggestion
# Splits a list of paths joined by the OS-specific path separator.
```
Also what is meant by "a list of paths"? Isn't it just one path, split to it's components?